index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
+    <form id="survey-form">
+      <label for="name" id="name-label">
+        Name
+        <input id="name" type="text" placeholder="Enter your name" required
+      /></label>
+      <label for="email" id="email-label"
+        >Email
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Enter your email"
+          required
+      /></label>
+      <label for="number" id="number-label"
+        >Age (Optional)
+        <input id="number" type="number" min="10" max="99" placeholder="age" />
+      </label>
+      <select id="dropdown">
+        <option value="">Student</option>
+        <option value="">Full time Job</option>
+        <option value="">Full Time Learner</option>
+        <option value="">prefered not to say</option>
+        <option value="">other</option>
+      </select>
+
+      <label for="recommend"
+        ><input type="radio" name="recommend" value="1" /> Definitely</label
+      >
+      <label for="recommend"
+        ><input type="radio" name="recommend" value="2" /> Maybe</label
+      >
+      <label for="recommend"
+        ><input type="radio" name="recommend" value="3" /> Not sure</label
+      >
+      <label for=""
+        ><input type="checkbox" value="Front-end" /> Front-end Projects</label
+      >
+
+      <label for=""
+        ><input type="checkbox" value="Back-end" /> Back-end Projects</label
+      >
+
+      <label for=""
+        ><input type="checkbox" value="data" /> Data Visualization
+      </label>
+
+      <label for=""
+        ><input type="checkbox" value="challenge" /> Challenge</label
+      >
+      <label for=""
+        ><input type="checkbox" value="data" /> Data Visualization</label
+      >
+      <label for=""
+        ><input type="checkbox" value="open" /> Open Source Community</label
```

Comment:
Please note that it is always a good idea to include a "For" property for labels. This will increase the clickable area for the users, hence improving UI. I noticed that even though some labels have this property, some do not.